src/openforms/ui/tests/test_header.py

```diff
@@ -0,0 +1,28 @@
+from unittest.mock import patch
+
+from django.urls import reverse
+
+from django_webtest import WebTest
+
+from openforms.config.models import GlobalConfiguration
+from openforms.forms.tests.factories import FormFactory
+
+
+class HeaderTest(WebTest):
+    @patch("openforms.config.models.GlobalConfiguration.get_solo")
+    def test_logo_label(self, get_solo_mock):
+        get_solo_mock.return_value = GlobalConfiguration(organization_name="Orgy")
+        form = FormFactory.create()
+        url = reverse("forms:form-detail", kwargs={"slug": form.slug})
+
+        response = self.app.get(url)
+
+        self.assertEqual(200, response.status_code)
+
+        header_node = response.html.find("header")
+        link_node = header_node.find("a")
+        aria_label = link_node.get("aria-label")
+        title = link_node.get("title")
+
+        self.assertEqual(aria_label, "Back to website of Orgy")
+        self.assertEqual(title, "Back to website of Orgy")
```

Comment:
you _must_ always use the translatable strings in tests with the correct interpolation format, as these tests will break when we actually enter the translations.